troposphere/static/js/components/admin/ResourceMaster.jsx

```diff
@@ -51,7 +50,7 @@ const ResourceMaster = React.createClass({
 
     onSelect(request) {
         // Navigate to a request at the top of the list
-        browserHistory.push(`/application/admin/resource-requests/${request.id}`);
+        this.props.router.push(`/admin/resource-requests/${request.id}`);
```

Comment:
Connor knows this - but for folks reading and observing ... or, _for inquiring minds_ - this change leverages that `withRouter` knows the basename for Troposphere is `/application`